index.html

```diff
@@ -74,14 +74,17 @@
           <a class="nav-link" href="#tutorials">Tutorials</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link" href="#developmentstatus">Development status</a>
+          <a class="nav-link" href="#developmentstatus">Development</a>
         </li>
         <li class="nav-item">
           <a class="nav-link" href="faq.html">Frequently Asked Questions</a>
```

Comment:
Also need to abbreviate to FAQ to fit in the header?
```suggestion
          <a class="nav-link" href="faq.html">FAQ</a>
```

Reply:
that would help indeed - when I checked locally, the navbar was a bit off and the change in 'Development' was sufficient. But if it's okay for you to make the shift to FAQ, then that would be great for the navbar.